packages/config/src/layer2s/bobanetwork.ts

```diff
@@ -1,57 +1,26 @@
-import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/shared-pure'
+import { assert, UnixTime } from '@l2beat/shared-pure'
 
-import {
-  addSentimentToDataAvailability,
-  CONTRACTS,
-  EXITS,
-  FORCE_TRANSACTIONS,
-  makeBridgeCompatible,
-  NUGGETS,
-  OPERATOR,
-  RISK_VIEW,
-  TECHNOLOGY_DATA_AVAILABILITY,
-} from '../common'
+import { NUGGETS } from '../common'
 import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
-import { OPTIMISTIC_ROLLUP_STATE_UPDATES_WARNING } from './common/liveness'
-import { getStage } from './common/stages/getStage'
+import { opStackL2 } from './templates/opStack'
 import { Layer2 } from './types'
 
 const discovery = new ProjectDiscovery('bobanetwork')
-
-const upgradesProxy = {
-  upgradableBy: ['Owner'],
-  upgradeDelay: 'No delay',
-}
-
-const upgradesAddressManager = {
-  upgradableBy: ['Owner'],
+const upgradeability = {
```

Comment:
given this PR, this is not needed anymore: https://github.com/l2beat/l2beat/pull/3421

please check if there is anything else that can be simplified

Reply:
nice, only little problem is that the L1ERC721Bridge does not show the upgradability then..

Reply:
ideal would be if it could also be added to the template (as optional if possible)